xir/validator.py

```diff
@@ -246,11 +246,11 @@ def _check_statement_applications(
         Checks the remaining conditions documented (but not implemented) by
         :func:`Validator._check_statements()`.
         """
-        # check that only integer wire labels are used at a script level
+        # check that only integer wire labels are used at a script-level
         if declared_params is None and any(isinstance(wire, str) for wire in stmt.wires):
             msg = (
                 f"Statement '{stmt}' is applied to named wires. Only integer wire labels are "
-                "allowed at a script level."
+                "allowed at a script-level."
```

Comment:
This doesn't really matter, but is it correct to put a hyphen here? My [understanding](https://www.grammarly.com/blog/hyphen/) is that "script-level statements" requires a hyphen but "at script level" doesn't.